index.html

```diff
@@ -26,104 +26,431 @@
             </div>
           </div>
-          <div>
-            <div class="sh-grp-item">
-              <svg class="sh-grp-three" width="75" height="75" viewBox="0 0 75 75" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <circle cx="37.5" cy="37.5" r="37" fill="white" stroke="#EBEBFF"/>
-                </svg>              
+          <div class="languages flex-col">
+            <button type="button" class="collapse-btn">Languages</button>
+            <div class="languages-content">
+              <div class="langs-container flex">
+                <div class="language-item flex">
+                  <img src="./assets/js.png" alt="javascript" class="img-lang">
+                  <div class="language-text">Javascript</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/html.png" alt="javascript" class="img-lang">
+                  <div class="language-text">HTML</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/css.png" alt="javascript" class="img-lang">
+                  <div class="language-text">CSS</div>
+                </div>
+              </div>
+            </div>
+            <button type="button" class="collapse-btn">Frameworks</button>
+            <div class="languages-content">
+              <div class="langs-container flex">
+                <div class="language-item flex">
+                  <img src="./assets/js.png" alt="javascript" class="img-lang">
+                  <div class="language-text">Javascript</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/html.png" alt="javascript" class="img-lang">
+                  <div class="language-text">HTML</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/css.png" alt="javascript" class="img-lang">
+                  <div class="language-text">CSS</div>
+                </div>
+              </div>
+            </div>
+            <button type="button" class="collapse-btn">Skills</button>
+            <div class="languages-content">
+              <div class="langs-container flex">
+                <div class="language-item flex">
+                  <img src="./assets/js.png" alt="javascript" class="img-lang">
+                  <div class="language-text">Javascript</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/html.png" alt="javascript" class="img-lang">
+                  <div class="language-text">HTML</div>
+                </div>
+                <div class="language-item flex">
+                  <img src="./assets/css.png" alt="javascript" class="img-lang">
+                  <div class="language-text">CSS</div>
+                </div>
+              </div>
```

Comment:
- [ ] The buttons "Languages", "Frameworks" and "Skills" should be in `<ul>` tags while their respective 'Javascript', 'HTML' and 'CSS' children should be in `<li>` tags.
- [ ] Please stick to the design template as much as possible:

This is the template for ABOUT MYSELF section is this:

![nta-about-orig](https://user-images.githubusercontent.com/36542986/133858802-e3aabd90-c72b-4bc7-89d6-4cae683f3713.jpg)

But your design looks like so:

![nta-about-new](https://user-images.githubusercontent.com/36542986/133858866-444368d9-46e8-4369-a7f0-c42b5c71103d.jpg)